luminosityleds/client
# LL-259

# Description

Used React Context to ensure 'lightmode' state persists through browser actions and navigating to other areas of the site. Also added QoL changes for run commands.

Subsequent # (issue)
When navigating away-from and back-to the home page, the state of 'light/dark mode' would not persist. This was also true for other actions taken in the browser.
## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

After implementation, I navigated to and from the home page while checking the state of the 'light/dark mode' to make sure it was consistent with whichever mode I had selected. I also exited and reopened the site to make sure that my selection was still consistent.

1. npm run dev
2. Change light toggle
3. Go to Login Page
4. Go to Home Page by clicking Luminosity LEDs in the upper left
5. Make sure the 'light/dark mode' is consistent, and persists through these actions.

# Checklist (when relevant):

      [TypeScript](https://google.github.io/styleguide/tsguide.html)
      [Schema](<https://learn.microsoft.com/en-us/previous-versions/windows/desktop/forefront-2010/ee652242(v=vs.100)>)
- [x] I have included a Aaron, Alex, Edgar, and Nick on the pull request
